map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -206,6 +209,16 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
                             .build()
             ))
 
+            map.setOnMapClickListener {
+                val objectDetailsFragment = activity?.supportFragmentManager?.findFragmentByTag(OBJECT_DETILAS)
```

Comment:
make onFocusLost() method in MapViewModel, business logic like this should be handled by that

Reply:
also use `requireActivity()`. theres no need to null safe the `activity` as it should exist when its called.